pkg/ss/partition/fa.go

```diff
@@ -18,9 +18,13 @@ func (fa *followingActor) become() error {
 			pa := &primaryActor{fa.actorBase}
 			pa.setState(Primary)
 			return pa.become()
-		case common.ClientMsg:
-			// we needn't handle this because primary would've updated the state.
-			// todo what should we do when primary is in different node?
+		case *common.ProxiedMsg:
+			// todo it looks a bit odd for follower to process client messages, but this is ensuring
+			// the correctness of algorithm. we can revisit this.
+			pm := m.(*common.ProxiedMsg)
+			handleClientMsg(&fa.actorBase, pm.ClientMsg)
+		case *common.ReplicatedMsg:
+			handleReplicatedMsg(&fa.actorBase, m.(*common.ReplicatedMsg))
 		default:
 			fa.logger.Warn("not handled", zap.Int64("part", fa.id), zap.Any("state", fa.getState().String()), zap.Any("type", reflect.TypeOf(m)))
```

Comment:
this is data plane were logs even if enabled, but will be sampled and throttled. Always emit metrics in data path

Reply:
commented out. added a todo to add metrics